scripts/jenkins/Jenkinsfile.release

```diff
@@ -337,6 +366,15 @@ def generatePublishStages() {
     params.fedora_revision
   )
 
+  tasks << publish(
+    'fedora-34',
+    DOCKER_IMAGES.fedora_34_installed.context,
+    'fedora-34',
+    'fedora-34',
+    release.&publishRpmPackages,
+    params.fedora_revision
+  )
```

Comment:
@robaerd you noted that RPM packages can be published in parallel. Parallel to other RPMs or parallel to DEBs?

Reply:
Both. The reason why DEBs can not be published in parallel with aptly is because they use a shared database. This is not the case with the RPMs as the repositories are isolated from another.

Reply:
Thank you! Sounds great, so basically what I did should work? 😄

Reply:
Yes, looks good!